app/src/main/java/org/mozilla/focus/customtabs/CustomTabsService.java

```diff
@@ -56,6 +56,11 @@ public int postMessage(ICustomTabsCallback iCustomTabsCallback, String s,
                                    Bundle bundle) throws RemoteException {
                 return 0;
             }
+
+            @Override
+            public boolean validateRelationship(ICustomTabsCallback iCustomTabsCallback, int i, Uri uri, Bundle bundle) throws RemoteException {
```

Comment:
I have no idea what this is @walkingice can you help take a look at this part?

Reply:
We can refer this API: https://developer.android.com/reference/android/support/customtabs/CustomTabsSession.html#validateRelationship(int,%20android.net.Uri,%20android.os.Bundle)

Since we don't support CustomTabs yet, return `false` looks good to me.